multimodal/src/autogluon/multimodal/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -122,6 +122,21 @@ model:
     image_size: 224
     max_img_num_per_col: 2
 
+  mmocr_text_detection:
+    checkpoint_name: "TextSnake"
+    data_types:
+      - "image"
+    train_transform_types:
+      - "resize_shorter_side"
+      - "center_crop"
+      - "trivial_augment"
+    val_transform_types:
+      - "resize_shorter_side"
+      - "center_crop"
+    image_norm: "imagenet"
+    image_size: 224
+    max_img_num_per_col: 2
```

Comment:
What will happen if the column contains multiple images?

Reply:
The current version only supports single-image inference, and I will add support for multiple-image inference later. But preprocessing of the image in autogluon/multimodal seems to be only for a single image, that is, the process_one_sample function is called. So a column only contains one image, I don't know if my understanding is correct.

Reply:
Yes, right now we only call `process_one_sample` function, so it doesn't support multi-image inference yet, will add in the next PR.